go/v1/statement_test.go

```diff
@@ -84,7 +84,20 @@ func TestBadStatementSubject(t *testing.T) {
 
 	err = got.Validate()
 
-	assert.Error(t, err, "Error: created malformed Statement (bad subject)")
+	// With this input, we encounter the RD object's validation error
+	assert.ErrorIs(t, err, ErrRDRequiredField, "Error: created malformed Statement (bad subject)")
```

Comment:
nit: can the error message between these tests distinguish which field is the problem?  (This and the following test have the same messages).  Maybe resolved if they're separate tests?

Reply:
Done!